app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def splash_link
+    if current_user && current_user.recipient?
+      link_to "Profile", user_path, class: "btn btn-default btn-lg"
+    else
+      link_to "Donate", users_path, class: "btn btn-default btn-lg"
+    end
+  end
```

Comment:
This is a potential conditional to change what button is displayed on the splash page